dozuki.js

```diff
@@ -3,21 +3,24 @@
    function Dozuki(url, http) {
       baseUrl = url + "/api/2.0/";
       this.guides = {
-         get: function(guideid, langid, params) {
+         get: function(guideid, langid, params, headers) {
+            headers = headers || {};
             params = params || {};
             var url = baseUrl + "guides/" + guideid;
             if (langid) {
                params["langid"] = langid;
             }
 
+            if (!('X-ALLOW-HTTP' in headers)) {
+               headers['X-ALLOW-HTTP'] = 1;
```

Comment:
That makes sense, but I was just using what was already there - https://github.com/iFixit/dozuki-js/pull/2/files#diff-00ed2bdaa9827b181b106952aaaa4e87L19